packages/merge-on-green/src/merge-logic.ts

```diff
@@ -492,7 +492,7 @@ mergeOnGreen.commentOnPR = async function commentOnPR(
       owner,
       repo,
       issue_number: pr,
-      body: `Your ${owner}/${repo} PR ${pr} was not mergeable because either one of your required status checks failed, or one of your required reviews was not approved.`,
```

Comment:
I think it might be helpful to leave in which owner/repo/pr the messages are referring to where possible, unless there's a reason not to?

Reply:
my thinking was that the comment shows up on the given PR, on the given repo/organization, so thought it might feel redundant?

Reply:
you are totally right!